### DISCUSSION TITLE
Training data

### DISCUSSION THREAD

**Discussion by noncodo** [GoekeLab/m6anet#140] [Ideas]
Hello!
We have about ~300k sequins molecules in our RNA004 (promethION) data set. 
Would it be usefull for training (negative controls) and how to share it?

**Comment by noncodo** [comment]
Here is some quick initial data analysis <img width="589" alt="image" src="https://github.com/GoekeLab/m6anet/assets/4597999/ce903cd0-c281-4d93-9195-62fcde377698">

**Comment by jonathangoeke** [comment]
Thanks @noncodo! This might help to improve the RNA004 model if you can share the data for the sequins. I will send you details
